### PR TITLE
Get images

### PR DESCRIPTION
**Pull request by PedroAmorim**
Hi!

I would like to add a feature that gets all images from parsed URL.
What do you think about it?

Thanks!

### CODE REVIEW

**Comment by andreskrey** [comment]
Love it!

But I have two comments:

1) Would you mind addressing the issues in the tests? Travis is failing for all 5.* versions
2) Shouldn't we add a parameter or a key in the `$result` array to allow the user to retrieve all the images? This is going to change in v1 (result will be an object with getters instead of a dumb array), but at least we should give now an option to get this new piece of information.

**Comment by PedroAmorim** [comment]
Done! 
Sorry for the issues in the tests.
I also add a key in the $result as asked, and squash my new commits.

**Comment by andreskrey** [comment]
I'm nitpicking here but would you mind creating a new test function for the expected images array? The test name is `testHTMLParserParsesHTML` so we should do a new function to test the images function. You can use the same dataprovider.

Also, feel free to add yourself to the [AUTHORS.MD](https://github.com/andreskrey/readability.php/blob/master/AUTHORS.md) file.

**Comment by PedroAmorim** [comment]
Of course, it is not an issue!
What do you think about this 2 last commits?
If you agree I could squash them all in one commit.

File: src/HTMLParser.php [hunk 1]
```diff
@@ -510,6 +522,37 @@ public function getMainImage()
...
+        }
+
+        foreach ($this->dom->getElementsByTagName('img') as $img) {
+            if ($src = $img->getAttribute('src')) {
+                $result[] = $src;
+            }
+        }
+
+        if ($this->getConfig()->getOption('fixRelativeURLs')) {
+            list($pathBase, $scheme, $prePath) = $this->getPathInfo($this->getConfig()->getOption('originalURL'));
```
Comment (andreskrey) [inline src/HTMLParser.php]: This seems to be dead code. Those three vars are not used later in this function.

**Comment by andreskrey** [comment]
No need to squash those commits :)

File: src/HTMLParser.php (hunk 1 above)
Comment (PedroAmorim) [reply src/HTMLParser.php]: Thanks, fixed on commit
https://github.com/andreskrey/readability.php/pull/30/commits/4f0b60a3b39995d822359ea6e4fefa4b5751154e

**Comment by andreskrey** [comment]
Great job, thanks :)

**Pull request merged**